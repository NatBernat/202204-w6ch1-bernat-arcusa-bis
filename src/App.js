import ToDosPage from "./components/ToDosPage/ToDosPage";
import toDosList from "../data/data";

function App() {
  return (
    <div className="App">
      <ToDosPage toDosList={toDosList} />
    </div>
  );
}

export default App;
