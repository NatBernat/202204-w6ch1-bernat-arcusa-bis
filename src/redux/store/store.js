import { configureStore } from "@reduxjs/toolkit";
import { toDosReducer } from "./features/toDos/toDosSlice";

const store = configureStore({
  reducer: {
    toDos: toDosReducer,
  },
});

export default store;
