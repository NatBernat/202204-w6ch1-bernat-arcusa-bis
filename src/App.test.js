import TestRenderer from "react-test-renderer";
import { Provider } from "react-redux";
import App from "./App";
import store from "./redux/store/store";

describe("Given an App component", () => {
  describe("When rendered", () => {
    test("Then it should always match this snapshot", () => {
      const appComponent = TestRenderer.create(
        <Provider store={store}>
          <App />
        </Provider>
      );
      expect(appComponent).toMatchSnapshot();
    });
  });
});
