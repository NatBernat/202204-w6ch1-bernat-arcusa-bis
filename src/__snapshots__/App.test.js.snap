// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given an App component When rendered Then it should always match this snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Things To Do
  </h1>
  <form
    autoComplete="off"
    onSubmit={[Function]}
  >
    <label>
      New to do
      <input
        name="name"
        onChange={[Function]}
        type="text"
      />
    </label>
    <button
      type="submit"
    >
      Add
    </button>
  </form>
  <ul />
</div>
`;
